ion-schema/src/main/kotlin/com/amazon/ionschema/model/ValidValue.kt

```diff
@@ -0,0 +1,26 @@
+package com.amazon.ionschema.model
+
+import com.amazon.ion.IonValue
+
+/**
+ * Represents an argument to the `valid_values` constraint.
+ * @see [Constraint.ValidValues]
+ */
+sealed class ValidValue {
+    /**
+     * A single Ion value. May not be annotated.
```

Comment:
Do we need a note about the value the model holds and it is an Ion equivalence comparison?

Reply:
Maybe just like a warning to someone who is introspecting the model so they are clear on the semantics?

Reply:
> Do we need a note about the value the model holds and it is an Ion equivalence comparison?

Sure, I can add that. It is using Ion equivalence.